docs/index.md

```diff
@@ -1093,10 +1093,26 @@ Can be specified multiple times.
 
 Files having this extension will be considered test files. Defaults to `js`.
 
-Affects `--watch` behavior.
-
```

Comment:
These extensions still affect `--watch` behavior, right? When `--watch-files` is `undefined`, files in the `cwd` with extension `opts.extension` will be watched.

Reply:
I didn’t want to confuse user by mentioning that it may affect `--watch` here. The behavior is well documented in `--watch-files`. I can restore it if you want.

Reply:
Ok. Then we should also remove watch from the Yargs option description, which currently is:
`--extension          File extension(s) to load and/or watch`

Reply:
Fixed in https://github.com/mochajs/mocha/pull/3980/commits/96c69f8ae17a5f9e6f57f69fc88bfd148de0e9f8